styles.css

```
@@ -139,17 +141,12 @@ nav .right-content a:hover {
     justify-content: center
 }
 
-.text-button{
-    font-size: 21px;
-    font-weight: bolder;
-}
-
 #container-buttons a{
     text-decoration: none;
     color: #000000;  
 }
 
-a :hover{
+#container-buttons a:hover{
     background-color: #F3F3F3;
```

```suggestion
    background-color: transparent;
```
Imagine que em algum momento desejamos trocar o background de toda a pagina para vermelho. Caso isso acontecesse voce teria que mudar o background do seu botao tb. O que acha do mudarmos o  background para transparent quando passarmos o mouse por cima? Dessa forma o cor de fundo do botao vai sempre o mesmo que o fundo da pagina.

---

Sugestão adicionada.